relayer/pkg/chainlink/ocr2/config_digester.go

```diff
@@ -7,7 +7,6 @@ import (
 	"strings"
 
 	"github.com/NethermindEth/juno/pkg/crypto/pedersen"
-	"github.com/NethermindEth/juno/pkg/rpc"
```

Comment:
note: removing this `juno/pkg/rpc` cleaned up a lot of dependencies and also fixed some build contraints - it's also just simpler types now